Source/AaxDecrypter/AaxDecrypter.csproj

```diff
@@ -5,8 +5,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="AAXClean" Version="0.4.7" />
-    <PackageReference Include="AAXClean.Codecs" Version="0.2.7" />
+    <PackageReference Include="AAXClean.Codecs" Version="0.2.9" />
```

Comment:
Cool. How did you manage to deploy these together?

Reply:
I didn't, but I never needed to include AAXClean independently anyway since it's already referenced by AAXClean.codecs.

Reply:
Forehead smack. Of course. I'm so not on top of my game today